api/envoy/config/core/v3/protocol.proto

```diff
@@ -214,6 +214,7 @@ message HttpProtocolOptions {
   // Action to take when a client request with a header name containing underscore characters is received.
   // If this setting is not specified, the value defaults to ALLOW.
   // Note: upstream responses are not affected by this setting.
```

Comment:
Maybe just also clarify that this configuration specified at the cluster is ignored.